tdrs-backend/tdpservice/search_indexes/parsers/tanf_parser.py

```diff
@@ -95,3 +97,15 @@ def parse(datafile):
         else:
             logger.warn("Parsing for type %s not yet implemented", record_type)
             continue
+
+def validate(family_case_schema, t1):
+    """Validate the datafile."""
+    errors = []
+    for field in family_case_schema.get_all_fields():
+        if len(field.validators) > 0:
```

Comment:
You don't need a length check here, the next line iterates through the list. On an empty list, it doesn't run the loop.